namer/marathon/src/test/scala/io/buoyant/namer/marathon/MarathonTest.scala

```diff
@@ -37,6 +40,88 @@ class MarathonTest extends FunSuite {
     assert(marathon.uriPrefix.contains("/marathon"))
     assert(marathon._prefix.contains(Path.read("/io.l5d.marathon")))
     assert(marathon.ttlMs.contains(300))
+    assert(marathon.jitterMs.contains(60))
     assert(!marathon.disabled)
   }
+
+  test("parse config with default jitter") {
+    val yaml = s"""
+                  |kind:           io.l5d.marathon
+                  |prefix:         /io.l5d.marathon
+                  |host:           localhost
+                  |port:           80
+                  |uriPrefix:      /marathon
+                  |ttlMs:          300
+                  |useHealthCheck: false
+      """.stripMargin
+
+    val mapper = Parser.objectMapper(yaml, Iterable(Seq(MarathonInitializer)))
+    val marathon = mapper.readValue[NamerConfig](yaml).asInstanceOf[MarathonConfig]
+    assert(marathon.host.contains("localhost"))
+    assert(marathon.port.contains(Port(80)))
+    assert(marathon.uriPrefix.contains("/marathon"))
+    assert(marathon._prefix.contains(Path.read("/io.l5d.marathon")))
+    assert(marathon.ttlMs.contains(300))
+    assert(marathon.jitterMs.isEmpty)
+    assert(!marathon.disabled)
+  }
+
+  test("jitter TTLs with the default jitter") {
+    val yaml = s"""
+                  |kind:           io.l5d.marathon
+                  |prefix:         /io.l5d.marathon
+                  |host:           localhost
+                  |port:           80
+                  |uriPrefix:      /marathon
+                  |ttlMs:          300
+                  |useHealthCheck: false
+      """.stripMargin
+
+    val mapper = Parser.objectMapper(yaml, Iterable(Seq(MarathonInitializer)))
+    val marathon = mapper.readValue[NamerConfig](yaml).asInstanceOf[MarathonConfig]
+    val jitters = Stream.continually(marathon.nextTtl)
+
+    val baseTtl = marathon.ttlMs.get.millis
+    val minTtl = baseTtl - 50.millis
+    val maxTtl = baseTtl + 50.millis
+
+    for (jitter <- jitters.take(300)) {
+      assert(jitter >= minTtl)
+      assert(jitter <= maxTtl)
+    }
+
+    // assert that we're generating a reasonable amount of distinct TTLs
+    assert(jitters.take(300).toSet.size >= 50)
```

Comment:
Changed this in 5d781156316f1ceb7d58e21a7fd8489c89e8aaaf. I took down the expected number of unique values to 2, and added a (perhaps too) long [comment](https://github.com/linkerd/linkerd/pull/1448/files#diff-99dfffd6beb2fa09f9d5c7dc5a7e9968R12) for historical reasons.

I think that seems fair?